src/main/java/io/netty/incubator/codec/quic/QuicConnectionAddress.java

```diff
@@ -69,8 +73,13 @@ public String toString() {
         if (this == EPHEMERAL) {
             return "QuicConnectionAddress{EPHEMERAL}";
         }
-        return "QuicConnectionAddress{" +
-                "connId=" + connId + '}';
+        ByteBuf buffer = Unpooled.wrappedBuffer(connId);
+        try {
+            return "QuicConnectionAddress{" +
+                    "connId=" + ByteBufUtil.hexDump(buffer) + '}';
```

Comment:
I'm using `LoggingHandler` where `toString` is invoked on every event. Is it possible to cache this transformation?